src/signalcorr.jl

```diff
@@ -56,6 +56,8 @@ before computing the autocovariance.
 If `x` is a vector, `r` must be a vector of the same length as `x`.
 If `x` is a matrix, `r` must be a matrix of size `(length(lags), size(x,2))`, and
 where each column in the result will correspond to a column in `x`.
+
+The output is normalized by `size(x, 1)`. See `autocor!` for normalization including the variance.
```

Comment:
I don't think the first sentence should be there. It would correspond to saying that the sample mean is scaled by the sample size.  Maybe something like "See `autocor!` for a version normalized with the variance." Similar comments apply to the other docstrings.